samples/README.md

```diff
@@ -11,4 +11,5 @@ There are also longer scripts that can be useful:
 |[dcat.sh](dcat.sh)|Cat sequential datasets or PDS members (supports wildcards in dataset and member)|
 |[dump_and_filter_racf.sh](dump_and_filter_racf.sh)|Dump and Filter RACF database for two record types|
 |[rcvptf.sh](rcvptf.sh)|Receive a PTF that you have uploaded to the Unix System Services zFS file system from ShopZ.|
-|[zcx_versions.py](zcx_versions.py)|Check running zCX instances to see if any can be upgraded.|
+|[zcx_versions.py](zcx_versions.py)|Check running zCX instances to see if any can be upgraded.|[smpe_list.py]smpe_list.py|Sample code showing how to convert from JCL to Python using the list feature of SMPE.|
```

Comment:
This should be broken into two lines. Also the second link is broken:

This:
|[zcx_versions.py](zcx_versions.py)|Check running zCX instances to see if any can be upgraded.|[smpe_list.py]smpe_list.py|Sample code showing how to convert from JCL to Python using the list feature of SMPE.|

Should be this:
|[zcx_versions.py](zcx_versions.py)|Check running zCX instances to see if any can be upgraded.|
|[smpe_list.py](smpe_list.py)|Sample code showing how to convert from JCL to Python using the list feature of SMPE.|

The fix:
```
|[zcx_versions.py](zcx_versions.py)|Check running zCX instances to see if any can be upgraded.|
|[smpe_list.py](smpe_list.py)|Sample code showing how to convert from JCL to Python using the list feature of SMPE.|
```